### ISSUE TITLE
iperf3 cannot stop in udp mode

### ISSUE THREAD

**Issue by qhsong** [esnet/iperf#433]
Hi everyone, 
Recently I use UDP test to test my speed, It often hang in last interval unless I press `^C`,
The command is `iperf3 -c host-2 -M 1300 -u -V`
the result like this:

``` bash
iperf 3.0.7
Linux host-1 3.16.0-4-amd64 #1 SMP Debian 3.16.7-ckt20-1+deb8u2 (2016-01-02) x86_64 GNU/Linux
Time: Wed, 08 Jun 2016 06:47:00 GMT
Connecting to host host-2, port 5201
      Cookie: host-1.1465368419.846682.7857a98a575
[  4] local 2001:da8:215:8e9::2 port 50354 connected to 2001:da8:215:27fe::2 port 5201
Starting Test: protocol: UDP, 1 streams, 8192 byte blocks, omitting 0 seconds, 10 second test
[ ID] Interval           Transfer     Bandwidth       Total Datagrams
[  4]   0.00-1.00   sec   120 KBytes   983 Kbits/sec  15
[  4]   1.00-2.00   sec   128 KBytes  1.05 Mbits/sec  16
[  4]   2.00-3.00   sec   128 KBytes  1.05 Mbits/sec  16
[  4]   3.00-4.00   sec   128 KBytes  1.05 Mbits/sec  16
[  4]   4.00-5.00   sec   128 KBytes  1.05 Mbits/sec  16
[  4]   5.00-6.00   sec   128 KBytes  1.05 Mbits/sec  16
[  4]   6.00-7.00   sec   128 KBytes  1.05 Mbits/sec  16
[  4]   7.00-8.00   sec   128 KBytes  1.05 Mbits/sec  16
[  4]   8.00-9.00   sec   128 KBytes  1.05 Mbits/sec  16
^C[  4]  10.00-15.87  sec  0.00 Bytes  0.00 bits/sec  0
- - - - - - - - - - - - - - - - - - - - - - - - -
Test Complete. Summary Results:
[ ID] Interval           Transfer     Bandwidth       Jitter    Lost/Total Datagrams
[  4]   0.00-15.87  sec  1.24 MBytes   657 Kbits/sec  0.000 ms  0/159 (0%)
[  4] Sent 159 datagrams
CPU Utilization: local/sender 36.1% (9.2%u/26.9%s), remote/receiver 0.0% (0.0%u/0.0%s)
iperf3: interrupt - the client has terminated
```

After I use `tcpdump` to capture the packets, I find client didn't send `FIN` packet, so the iperf3 client hang in last packet, and after I press `^C`, it wil send FIN packet and got some `RST` packets.

The TCP test work fine but UDP work not well, is a bug?Or I use it in wrong mode.

**Comment by qhsong** [comment]
server output like this:
```bash
-----------------------------------------------------------
Server listening on 5201
-----------------------------------------------------------
Accepted connection from , port 38254
[  5] local  port 5201 connected to  port 46066
^C[ ID] Interval           Transfer     Bandwidth       Jitter    Lost/Total Datagrams
[  5]   0.00-14.57  sec  0.00 Bytes  0.00 bits/sec  0.000 ms  0/0 (-nan%)
- - - - - - - - - - - - - - - - - - - - - - - - -
[ ID] Interval           Transfer     Bandwidth       Jitter    Lost/Total Datagrams
[  5]   0.00-14.57  sec  0.00 Bytes  0.00 bits/sec  0.000 ms  0/0 (-nan%)
iperf3: interrupt - the server has terminated
```

**Comment by BharatNT2IE** [comment]
You are atleast able to initiate :) . I am not even able to start :(  . Are you using a Ubuntu Machine...

**Comment by qhsong** [comment]
I use Debian, seem ipref3 have some problems in UDP test.

**Comment by bmah888** [comment]
We believe this was fixed as a result of packet size fixes in iperf-3.1.5.  Can you retest with at least 3.1.5 or newer, and see if this is still happening?

**Comment by bltierney** [comment]
closing. Please reopen if necessary
